ai_edge_torch/generative/layers/model_config.py

```diff
@@ -55,9 +55,10 @@ class FeedForwardType(enum.Enum):
 
 @dataclass
 class AttentionConfig:
-  """Attention moduel's parameters."""
+  """Attention model's parameters."""
```

Comment:
I think `module` is probably the best choice and follows PyTorch naming convention here. Both are better than the existing `moduel` though :-)

Reply:
+1